internal/autotag/performer_test.go

```diff
@@ -275,7 +276,8 @@ func testPerformerGalleries(t *testing.T, performerName, expectedRegex string) {
 		Direction: &direction,
 	}
 
-	db.Gallery.On("Query", mock.Anything, expectedGalleryFilter, expectedFindFilter).Return(galleries, len(galleries), nil).Once()
+	db.Gallery.On("Query", mock.Anything, gallery.QueryOptions(expectedGalleryFilter, expectedFindFilter, false)).
```

Comment:
Only related to this method definition change: https://github.com/stashapp/stash/pull/4423/files?diff=split&w=1#r1443579818
No change in behavior expected.